### ISSUE TITLE
Uncaught TypeError: Cannot read property 'apply' of undefined

### ISSUE THREAD

**Issue by tgolen** [casesandberg/reactcss#26]
![image](https://cloud.githubusercontent.com/assets/1228807/12053696/c6b8704c-aed8-11e5-9560-21677b15c57f.png)

This happens when I try to implement your React Color picker, but it looks to be related to this lib. For some reason, the `merge` library isn't being included properly and I am having a hard time figuring out why.

I am using webpack, and just trying to use a simple component:

```js
var React = require('react');
var ColorPicker = require('react-color');

export default class Component extends React.Component {

  render() {
    return <ColorPicker type="sketch" />;
  }
}
```

When `merge=require("merge");` is ran from `lib/merge.js`, all that is imported is an empty object. When I log the imported module to the console, all I see is `Object {}`. It seems to me that it's getting the `merge` module from somewhere other than the NPM dependency.

**Comment by casesandberg** [comment]
What does your webpack settings look like? Also what version of `react-color` are you using?

**Comment by relifeted** [comment]
I have the same problem, seems that webpack confused when resolving the module "merge".

I try to rename merge.js to other name, and also change require syntax, it works.

**Comment by casesandberg** [comment]
This was fixed in `reactcss@0.4.4`
